packages/devtools_app/lib/src/memory/flutter/memory_screen.dart

```diff
@@ -78,6 +78,69 @@ class MemoryBodyState extends State<MemoryBody> {
     );
   }
 
+  static const String _liveFeed = 'Live Feed';
+  String memorySource = _liveFeed;
+
+  Widget createMenuItem(String name) {
+    final rowChildren = memorySource == name
+        ? [
+            Icon(Icons.check, size: 12),
+            const SizedBox(width: 10),
+            Text(name),
+          ]
+        : [
+            const SizedBox(width: 22),
+            Text(name),
+          ];
+
+    return PopupMenuItem<String>(
+      value: name,
+      child: Row(
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: rowChildren,
+      ),
+    );
+  }
+
+  Widget _selectMemoryFile() {
```

Comment:
this method name sounds like it is responsible for selecting a memory file. How about `memorySourceDropdown` or similar?

Reply:
Done.